app/workers/update_schedule_worker.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateScheduleWorker
+  include Sidekiq::Worker
+
+  def self.perform
+    Student.all.each do |student|
+      student.update(at_school: false)
+      UpdateScheduleJob.perform(student)
```

Comment:
then they would be in the lost bin and teachers can then go and manually mark them absent.